controllers/article-controller.js

```diff
@@ -12,7 +12,22 @@ exports.getArticleById = (req, res, next) => {
 }
 
 exports.getAllArticles = (req, res, next) => {
-    fetchAllArticles()
+    let { sort_by, order } = req.query;
```

Comment:
We don't want these variables to be redeclared, so they should be defined with `const`